README.md

```diff
@@ -12,6 +12,26 @@ This package focuses on eight widely used regressor models, providing a curated
 
 ## Installation
 
+Since this package is still in the developing process, it has not been published on PyPI yet. Thus, In order to use this package, please run the instructions provided below.
+
+### Initialization
+1. Clone this GitHub repository using this command:
```

Comment:
Optional:
We might want to add a note about where the command should be run > terminal or or command prompt/Git Bash.